Xamarin.Essentials/Accelerometer/Accelerometer.android.cs

```diff
@@ -45,6 +45,9 @@ void ISensorEventListener.OnAccuracyChanged(Sensor sensor, [GeneratedEnum] Senso
 
         void ISensorEventListener.OnSensorChanged(SensorEvent e)
         {
+            if (e?.Values?.Count < 3)
```

Comment:
Is null < 3? If we are sure we will get 3 values, then maybe != 3?

Reply:
Doesn't null return the default? Which would be 0? Could test this out ;)